### PR TITLE
Resolve Eslint errors

### PR DESCRIPTION
**Pull request by NCMoseley**

### CODE REVIEW

File: js/screens/EditProfile/EditProfile.js [hunk 1]
```diff
@@ -15,30 +16,6 @@ import Button from "../../components/Button";
...
-  const task = firebaseRef.child(name).put(file, metadata);
-  task
-    .then(s => {
-      const url = s.downloadURL;
-
-      console.log(url);
-    })
-    .catch(error => error.message);
-};
 // This is a work in progress
```
Comment (siddharthparmar7) [inline js/screens/EditProfile/EditProfile.js]: Please delete this comment.

File: js/navigation/TabNav/stacks/TeamsStack.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 //todo : replace this with the Teams/messaging stack when complete next week. requires a placeholder for now.
```
Comment (siddharthparmar7) [inline js/navigation/TabNav/stacks/TeamsStack.js]: Please delete this comment.

File: js/screens/GroupDetails/GroupDetailsContainer.js [hunk 3]
```diff
@@ -14,33 +13,33 @@ class GroupDetailsContainer extends Component {
...
   componentDidMount() {
     // TODO: Change this param to dynamic, once navigation is set up
-    const param = "";
-
-    this.asyncFetchUserList(param);
+    this.asyncFetchUserList();
   }
   render() {
-    console.log(this.state.list);
+    // console.log(this.state.list);
```
Comment (siddharthparmar7) [inline js/screens/GroupDetails/GroupDetailsContainer.js]: remove this comment.

File: js/screens/GroupDetails/GroupDetailsContainer.js [hunk 4]
```diff
@@ -14,33 +13,33 @@ class GroupDetailsContainer extends Component {
...
         });
       })
-      .catch(err => console.log("query user error: ", err));
+      .catch(err => {
+        console.log("Error getting documents", err);
+      });
   }
 
   componentDidMount() {
     // TODO: Change this param to dynamic, once navigation is set up
```
Comment (siddharthparmar7) [inline js/screens/GroupDetails/GroupDetailsContainer.js]: remove this comment, please

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Good work, Nate! Thank you and please make the requested changes so I can approve this pull request.

**Comment by NCMoseley** [comment]
Please review

**Review by siddharthparmar7 — APPROVED** [review]
Great! Thank you, Nate!

**Pull request merged**